packages/legend-studio/src/components/editor/edit-panel/connection-editor/RelationalDatabaseConnectionEditor.tsx

```diff
@@ -106,15 +78,49 @@ import {
   snowflakePublicAuthenticationStrategy_setPassPhraseVaultReference,
   snowflakePublicAuthenticationStrategy_setPrivateKeyVaultReference,
   snowflakePublicAuthenticationStrategy_setPublicUserName,
+  spannerDatasourceSpecification_setDatabaseId,
+  spannerDatasourceSpecification_setHost,
+  spannerDatasourceSpecification_setInstanceId,
+  spannerDatasourceSpecification_setPort,
+  spannerDatasourceSpecification_setProjectId,
   staticDatasourceSpecification_setDatabaseName,
   staticDatasourceSpecification_setHost,
   staticDatasourceSpecification_setPort,
   usernamePasswordAuthenticationStrategy_setBaseVaultReference,
   usernamePasswordAuthenticationStrategy_setPasswordVaultReference,
   usernamePasswordAuthenticationStrategy_setUserNameVaultReference,
-  gcpWorkloadIdentityFederationAuthenticationStrategy_setServiceAccountEmail,
-  gcpWorkloadIdentityFederationAuthenticationStrategy_setAdditionalGcpScopes,
 } from '../../../../stores/graphModifier/StoreRelational_GraphModifierHelper.js';
+import {
```

Comment:
please move this import back where it was in the code

Reply:
ok, but take into consideration that this is what IntelliJ autoimport does. If you have better a codestyle pls share, otherwise it would be hard to maintain code cause autoformat would not be possible.

Reply:
I see, we're not aware of this, we're using VSCode majorly for our development.